pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -300,6 +315,36 @@ func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.
 	return pluginToNodeScoreMap, nil
 }
 
+// RunNormalizeScorePlugins runs the normalize score plugins. It should be called after
+// RunScorePlugins with the PluginToNodeScoreMap result. It then modifies the map with
+// normalized scores. It returns a non-success Status if any of the normalize score plugins
+// returns a non-success status.
+func (f *framework) RunNormalizeScorePlugins(pc *PluginContext, pod *v1.Pod, scores PluginToNodeScoreMap) *Status {
+	ctx, cancel := context.WithCancel(context.Background())
+	errCh := schedutil.NewErrorChannel()
+	workqueue.ParallelizeUntil(ctx, 16, len(f.normalizeScorePlugins), func(index int) {
+		pl := f.normalizeScorePlugins[index]
+		nodeScoreList, ok := scores[pl.Name()]
```

Comment:
We could precheck it when initializing the NormalizeSocrePlugins.

Reply:
If this is a requirement for running normalizes extension point, we could embed ScorePlugin into NormalizeScorePlugin.

Reply:
I will leave this open for more inputs. I think precheck is good but I am not sure if we should fail the initialization or just print some warning message. 

The current implementation assumes there isn't a necessary match of score plugin list and normalize score plugin list, and it doesn't fail

@salamat, what do you think?

Reply:
@bsalamat  has clarified this in [this comment](https://github.com/kubernetes/kubernetes/pull/80383#discussion_r306044447). I am going to add precheck logic to make sure that the list of enabled Normalize Score extension points is a subset of Score extension points. And hence the check here is not needed.